Algorithm/Number_theory/readme.md

```diff
@@ -36,6 +36,5 @@
 <hr>
 
 
-# Questions related to Number Theory
-
```

Comment:
you are strictly not allowed to delete any line written before.
Revert changes asap else your pr will be closed